### DISCUSSION TITLE
Trigger a flow from HA lovelace button

### DISCUSSION THREAD

**Discussion by marc965** [hassio-addons/addon-node-red#1319] [Q&A] [answered]
How can i trigger a node-red flow with a lovelace button with this node-red addon only?
I only found some solutions with this integration: [zachowj/hass-node-red]

Is this available with this official addon too?

**Comment by frenck** [comment, marked as answer]
The integration can be used with the add-on. However it is not a requirement.
You could also create an input_button entity or button helper in Home Assistant to trigger an event.

**Comment by marc965** [comment]
ok. thanks, i try this weekend...

**Reply by marc965** [reply]
fantastic, input_button is exactly what i am looking for, thank you very much, it works...
